### PR TITLE
Milestone 2: Works and About Me Sections

### PR DESCRIPTION
**Pull request by IndieCoderMM**
# Portfolio project milestone 2: Works and Aboutme Sections
In this milestone, I: 
- Added *Works* section using Grid & Flexbox
- Added info and skill list in *About me* section
- Added snapshots from the Figma template

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -64,5 +64,157 @@ <h1>I'm Hein Thant Glad to see you!</h1>
...
       </ul>
     </section>
+    <section class="works">
+      <section class="work-card">
+        <img
+          src="img/work-snapshot00.png"
+          alt="Webpage-snapshot"
+          class="work-snapshot"
+        />
+        <h1>Microverse Portfolio</h1>
```
Comment (jerryowusu) [inline index.html]: For SEO optimization and best standard practices, your website should have only one h1 tag. If you have more than one h1 tag on a page, change the other h1 tags to an h2 or h3. Check [this link](https://www.investisdigital.com/blog/seo/using-h1-tag-improves-search-engine-ranking#:~:text=To%20determine%20if%20you're,tags%20in%20the%20page%20code) for further explaination on that.

File: index.html [hunk 2]
```diff
@@ -64,5 +64,157 @@ <h1>I'm Hein Thant Glad to see you!</h1>
...
+      </section>
+    </section>
+    <section class="aboutme">
+      <h1>About Myself</h1>
+      <p>
+        Hello I'm a software developer! I can help you build a product , feature
+        or website Look through some of my work and experience! If you like what
+        you see and have a project you need coded, don't hestiate to contact me.
+      </p>
+      <p class="blue-text">LET'S CONNECT</p>
```
Comment (jerryowusu) [inline index.html]: This is a title so kindly consider using an h2-h6 tag instead of the paragraph

File: index.html [hunk 3]
```diff
@@ -64,5 +64,157 @@ <h1>I'm Hein Thant Glad to see you!</h1>
...
+        <ul class="work-tags">
+          <li>html</li>
+          <li>css</li>
+          <li>javaScript</li>
+        </ul>
+        <button class="btn see-project-btn">See Project</button>
+      </section>
+    </section>
+    <section class="aboutme">
+      <h1>About Myself</h1>
```
Comment (jerryowusu) [inline index.html]: As stated above, there should be only one h1 tag in the website. Kindly update accordingly

File: index.html [hunk 4]
```diff
@@ -64,5 +64,157 @@ <h1>I'm Hein Thant Glad to see you!</h1>
...
+          <a href="#">
+            <img src="img/icons/medium.png" alt="medium" class="social-icon" />
+          </a>
+        </li>
+      </ul>
+      <button class="btn get-resume-btn">Get my resume</button>
+      <ul class="skill-list">
+        <li class="languages">Languages <span class="arrow">></span></li>
+        <li class="frameworks">Frameworks <span class="arrow">></span></li>
+        <li class="skills">Skills <span class="arrow">></span></li>
```
Comment (jerryowusu) [inline index.html]: The language section is missing the sub-list of displayed language as shown in the design below. Kindly fix it.
Also ensure that the arrow to the right side of `Language` points down just like the figma design

Your design            |  The design template
:-------------------------:|:-------------------------:
Screenshot from 2022-10-27 10-10-32  |  Screenshot from 2022-10-27 10-10-20

**Review by jerryowusu — CHANGES_REQUESTED** [review]
Hi @IndieCoderMM 

Good job so far!
Your project is looking fantastic so far :star_struck: 

However, I've found some room for improvement that can help you accomplish a more readable/clean code and make your project look professional :sunglasses:

soclose2


## Highlights
All linters are passing :heavy_check_mark: 
Good commit messages :heavy_check_mark: 
Followed correct GitHub flow :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (Abenezer-Tilahun) [reply index.html]: Fixed.

File: index.html (hunk 2 above)
Comment (Abenezer-Tilahun) [reply index.html]: Fixed.

File: index.html (hunk 3 above)
Comment (Abenezer-Tilahun) [reply index.html]: Fixed.

File: index.html (hunk 4 above)
Comment (Abenezer-Tilahun) [reply index.html]: Fixed.

**Review by Abenezer-Tilahun — APPROVED** [review]
Hi @IndieCoderMM   
 
Your project is complete  👍 💯   There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 👏🏻

✔️ No linter errors.
✔️ You used correct Github Flow.
✔️ Professional README  
✔️ Good Commit Messages


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**